coderd/database/migrations/000021_template_autobuild_constraints.down.sql

```diff
@@ -1,2 +1,2 @@
-ALTER TABLE ONLY templates DROP COLUMN IF EXISTS max_ttl;
-ALTER TABLE ONLY templates DROP COLUMN IF EXISTS min_autostart_interval;
```

Comment:
Did you mean to change the database column names?

Reply:
Yes, I mean to only change the CLI flags.

Reply:
But the change in the database can also be done to avoid confusion.

Reply:
We cannot edit migrations, you have to make a new one to rename a column. That being said, our other columns use `_`.

Can you make a new commit to only affect the cli flags?

Reply:
ok